runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectRunnerTest.java

```diff
@@ -577,4 +628,11 @@ public void validate() {
       return underlying.getOutputCoder();
     }
   }
+
+  /**
+   * Variable holder for each test instance.
+   */
+  public static class Context {
```

Comment:
Sorry, what I meant was "Use `AtomicBoolean` instead of introducing a new `Context` class, as I can't imagine a situation that would expand this usage.

Reply:
Oh, was wondering what was the point. Will update the test tomorrow. Thanks for clarifying

Reply:
Ping?